openadapt/cache.py

```diff
@@ -34,5 +72,7 @@ def wrapper(*args, **kwargs):
             duration = time.time() - start_time
             logger.debug(f"{fn=} {duration=}")
             return rval
+
         return wrapper
-    return decorator
+
+    return decorator  # TODO: Implement the caching logic
```

Comment:
What is the TODO about? Isn't the caching logic implemented by this function?

Reply:
Currently in the main branch there is a TODO for the `cache` function, so I assume it should be removed.

Reply:
Addressed in https://github.com/OpenAdaptAI/OpenAdapt/pull/289/commits/d88767ff785961d7eb36ca589d96f707a019b507